auto_round/mllm/README.md

```diff
@@ -78,15 +68,17 @@ The llava calibration dataset supports the five existing MLLMs.
 
 ### Support Matrix
 
-The design of the MLLM model API is not uniform, and some models do not support the quantization nontext module. Quantization of the vision components of Llama-3.2-11B-Vision, Phi-3.5-vision-instruct and llava-v1.5-7b is currently supported.
+For existing MLLMs, the LLM components quantization is theoretically unproblematic. But, the design of vision components in MLLM model APIs is not standardized, and some models do not support the quantization of non-text modules.
```

Comment:
For typical VLLMs, we assume that the default quantization, which excludes quantizing the visual component, is supported.